packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -916,4 +916,29 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       uri: locusUrl,
     });
   }
+
+  /**
+   * Sends a request to set the step away (be right back) status.
+   *
+   * @param {Object} options - The options for the step away request.
+   * @param {boolean} options.enabled - Whether the step away status is enabled.
+   * @param {string} options.locusUrl - The URL of the locus.
+   * @param {string} options.deviceUrl - The URL of the device.
+   * @param {string} options.selfId - The ID of the participant.
+   * @returns {Promise}
+   */
+  getStepAway({enabled, locusUrl, deviceUrl, selfId}: StepAwayOptions) {
```

Comment:
@brycetham I feel `sendStepAway` or `stepAway` is better, too. I would go with `sendStepAway` because I see there are a lot of similar naming patterns in the `request.ts` file, and it clears action in this function better.